main.py

```diff
@@ -12,11 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Source code from https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/run/pubsub/main.py
+# The code in this module is based on https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/run/pubsub/main.py. See https://cloud.google.com/run/docs/tutorials/pubsub for the accompanying Cloud Run/PubSub solutions guide.
 
 # [START run_pubsub_server_setup]
 import base64
 import os
+import json
+from philips_hue import configure_light
```

Comment:
I think it's clearer if we just "import philips_hue"

So then the call is philips_hue.configure_light()?